contracts/identity/Identity.sol

```diff
@@ -38,12 +42,20 @@ contract Identity is IdentityAdminRole, SchemeGuard {
         onlyRegistered
         onlyIdentityAdmin
     {
-        claimers.add(account);
-        
-        increaseClaimerCount(1);
-        dateAdded[account] = now;
-        
-        emit ClaimerAdded(account);
+        _addClaimer(account);
+    }
+
+    function addClaimerWithDID(address account, string memory did) 
+        public
+        onlyRegistered
+        onlyIdentityAdmin 
+    {
+        _addClaimer(account);
+
+        bytes32 pHash = keccak256(bytes(did));
+        didLastAdded[did] = now;
```

Comment:
why not index by account?

Reply:
This was a mistake, as dateAdded already keeps track of when accounts were added. Will remove